pupa/cli/commands/update.py

```diff
@@ -232,7 +253,17 @@ def check_session_list(self, juris):
             raise CommandError('session(s) unaccounted for: %s' % ', '.join(unaccounted_sessions))
 
     def handle(self, args, other):
-        juris = self.get_jurisdiction(args.module)
+        juris, module = self.get_jurisdiction(args.module)
+        overrides = {}
+        overrides.update(getattr(module, 'settings', {}))
```

Comment:
Yep, we're overriding pupa settings with settings overrides from the scraper module and command-line arguments. This is meant to copy the logic in https://github.com/openstates/billy/blob/66c6588d96373f6874c4df006cf1b81a7bd72434/billy/bin/update.py#L275-L277.